### ISSUE TITLE
Presubmission feedback

### ISSUE THREAD

**Issue by PhilChodrow** [zhijianli9999/16bproject#2]
Overall, I think this is an excellent project! I enjoyed your presentation yesterday, and I also enjoyed reading your blog post just now. 

I think that a blog post is a totally fine way to communicate your findings in this data analysis project. 

Because you are comparing several different models and interpreting results, there are two components that I think would significantly add to the story: 

1. A measure of fit. Right now, you are describing model outputs "by eye" to see whether they "look like" they match observed trends. This isn't wrong, but you can do better. Can you add a relevant measure of fit that would let us quantify the accuracy of each model on the data? Maybe you could even do a train/test split to test your hypothesis about the ability of the model to make predictions? It's possible that the classes from PySAL will have measures of fit built in; if so, you can show and explain them. If not, you might want to use your own. [Cross-entropy](https://en.wikipedia.org/wiki/Cross_entropy#Cross-entropy_loss_function_and_logistic_regression) would be a simple and good one. 
2. Another thing you might consider adding are displays of coefficients. You mention that some coefficients are statistically significant in some models, which is great, but it would be best to show your reader the details! 

Nice work! I'm impressed with what you've done here.
